nodejs/nodejs.json

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "nodejs",
+  "display": "Node JS",
+  "run": {
+    "stop": "stop",
```

Comment:
Is this correct? In my opinion, a signal 2 (SIGINT) should be sent.

Reply:
Sorry, I just meant to "submit" FiveM. I still didn't finish NodeJS.
Also, I have no clues about signals.